Benchmarks/ReflectionPerf.cs

```diff
@@ -0,0 +1,65 @@
+﻿using BenchmarkComponent;
+using BenchmarkDotNet.Attributes;
+
+namespace Benchmarks
+{
+    [MemoryDiagnoser]
+    public class ReflectionPerf
+    {
+        ClassWithMarshalingRoutines instance;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            instance = new ClassWithMarshalingRoutines();
+        }
+
+        [Benchmark]
+        public object ExecuteComWrappersCreateKeyValuePairsFactoryWithNewObject()
```

Comment:
I would recommend renaming these functions to reflect user scenario and putting a comment if you want to indicate the exact function we are trying to benchmark in cswinrt which is what the function name reflects today.

Reply:
Something like ExecuteMarshalingForNewKeyValuePair or so.